common/scripted_triggers/imperia_aggressive_diplomatic_plays_permitted_custom.txt

```diff
@@ -1,12 +1,17 @@
-﻿imperia_aggressive_diplomatic_plays_permitted = {
-	#Country scope
-	AND = {
-		aggressive_diplomatic_plays_permitted = yes
-		NOT = {
-			has_modifier = no_more_war
-		}
-		NOT = {
-			has_law = law_type:law_true_neutrality
-		}
+﻿imperia_true_neutral_diplomatic_plays_permitted = {
```

Comment:
imperia_diplomatic_plays_permitted_relaxed. or something like that. Now it seems it is only affects true neutral countries.